docs/products/kafka/concepts/auth-types.rst

```diff
@@ -0,0 +1,73 @@
+Authentication types
+======================
+
+It is strongly recommended to use modern encryption protocols to protect data in transit sent to and from Apache Kafka.
+You can use one of the available options in **Aiven for Apache Kafka** services:
+
+Transport Layer Security
+------------------------
+
+**Transport Layer Security (TLS)**, also known as Secure Sockets
+Layer (SSL), is an established standard for securing internet traffic. This method
+relies on a certificate that is provided by a Certificate
+Authority (for example, `letsencrypt.org <http://letsencrypt.org>`_ ) for your domain.
+With this certificate and the right technical setup, you can use
+your domain to encrypt the traffic to your service.
+
+By default Aiven enables TLS encryption for all  **Aiven for Apache Kafka** services and helps with the application, renewal, and configuration of certificates.
+
+There are two ways you can use TLS:
+
+#. **TLS encryption** : your Apache Kafka client validates the certificate for
+   your Apache Kafka broker.
+
+#. **TLS authentication** : your Apache Kafka client validates the certificate
+   for your Apache Kafka broker and your broker validates the certificate
+   for your client.
+
+
+Simple Authentication and Security Layer
+----------------------------------------
+
+**Simple Authentication and Security Layer (SASL)** acts as a layer that
+allows alternative login methods for your service. For example, if you
+use Active Directory for authentication, SASL supports a Kerberos login
+method to provide access.
+
+We support SASL/PLAIN and SASL/SCRAM with **Aiven for Apache Kafka**.
+
+SASL/PLAIN
+~~~~~~~~~~
+
+**PLAIN** relies on a combination of username
+and password to log in over a TLS connection, meaning that your traffic
+is encrypted. You should not use SASL/PLAIN without TLS, because in this case
```

Comment:
should this be a warning?

Reply:
Having a warning is fine but there's no risk here because Aiven does not support non-TLS connections at all. (i.e. SASL_SSL protocol is supported, SASL_PLAINTEXT is not).

Reply:
Rephrased a bit. Could you check @dbeech if this reflects the reality?
 
I actually got here confused as well, because SASL/PLAIN  without TLS doesn't make any sense to me.  But after checking with Christopher Gwilliams we do have support for  SASL/PLAIN. And the content is based on the [help article ](https://help.aiven.io/en/articles/4331547-kafka-authentication-types)

Reply:
It is very confusing. There are two totally separate but similarly named things:
- The protocol, which can be `SASL_SSL` or `SASL_PLAINTEXT` (this is the part that dictates whether the communication link is encrypted)
- The authentication type, which can be SASL/PLAIN or SASL/SCRAM (this is the part that dictates how the credentials are handled, i.e. are we sending a plaintext password or an encrypted one)

When using SASL on Aiven, the user has no choice over the protocol - it's always `SASL_SSL`. They do get to choose the auth type `PLAIN` vs `SCRAM`, but even in the case of `PLAIN`, the "plain" creds are sent over an encrypted link, so the risk is somewhat mitigated.